warehouse/models/mart/gtfs/fct_daily_scheduled_stops.sql

```diff
@@ -0,0 +1,140 @@
+{{ config(materialized='table') }}
+
+WITH fct_daily_scheduled_trips AS (
+
+    SELECT *
+    FROM {{ ref('fct_daily_scheduled_trips') }}
+),
+
+dim_stop_times AS (
+    SELECT *
+    FROM {{ ref('dim_stop_times') }}
+),
+
+dim_stops AS (
+    SELECT *
+    FROM {{ ref('dim_stops') }}
+),
+
+dim_routes AS (
+    SELECT *
+    FROM {{ ref('dim_routes') }}
+),
+
+stops_by_day AS (
```

Comment:
I think it might be good to switch the order of this and the next CTE -- if you group by route type first, you can then just group that one to get daily total instead of grouping the full table. May want to double check that you get the same answer (are there any rows with null route_type? If so, may need to add handling for those, and in fact we may want to preserve them in the pivot too.)

If you do this, you'll need to add the `LOGICAL_OR` to the route type CTE as well.

Reply:
@lauriemerrell okay cool I just pushed a commit that reorders the CTEs, and the results appear the same. 

So there **are** null `route_type`s, maybe we should chat very quickly about the handling/preservation of them.